projects/rbenv.org/ruby-build/package.yml

```diff
@@ -0,0 +1,37 @@
+distributable:
+  url: git+https://github.com/rbenv/ruby-build.git
+  ref: ${{version.tag}}
+
+versions:
+  github: rbenv/ruby-build
+
+companions:
+  openssl.org: '*'
```

Comment:
This is a dynamic library and definitely is a required dependency and not a companion.

Reply:
well not that I looked, but companions are things that the package doesn't need to work. I feel it unlikely anything that needs openssl can work without it.